README.md

```diff
@@ -23,7 +23,7 @@ Ansible Role for installing the Docker runtime environment on your (Debian) host
 
 ## Requirements
 
-The role has been tested on Debian and is expected to work on Ubuntu.
+The role has been tested on Debian and Ubuntu 24 LTS target hosts.
```

Comment:
nit: Ubuntu 24.04 LTS, not Ubuntu 24.10.

Reply:
See 476042d7ddab7688691e67a040419cb5d08f69ca